lib/tapioca/internal.rb

```diff
@@ -1,6 +1,11 @@
 # typed: strict
 # frozen_string_literal: true
 
+# The rewriter needs to be loaded very early so RBS comments within Tapioca itself are rewritten
+require "spoom"
+require "tapioca/rbs/rewriter"
```

Comment:
We need this to happen super early so we also rewrite the RBS comments we find inside Tapioca